### PR TITLE
 Magic File Encoding v3.0.0

### PR DESCRIPTION
**Pull request by Jan5366x**
Will be merged after .Net 8.0 Release (November 2023) and it will bump the major version

### CODE REVIEW

**Comment by lg2de** [comment]
Hey @Jan5366x , the release is available now... ;)

**Comment by Jan5366x** [comment]
Hi @lg2de , fair point :smile: I plan to deploy v3.0.0 later this week.

**Comment by Jan5366x** [comment]
Hi @lg2de,

 v3.0.0 on .net 8.0 is now available on nuget.
 https://www.nuget.org/packages/MagicFileEncoding

I have postponed some planed (non-breaking) refactoring and improvements  to an upcoming version to bring out the .Net 8.0 dependency bump. :smile: 

Let me know if you encounter any problems.

Best regards

Jan

PS: Grüße aus Hamburg :wink:

**Comment by lg2de** [comment]
I wasn't waiting for this release. 2.0.1 is working fine also with my net8 project.
I was checking the status of the project before start using it.

File: global.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
-﻿{
+{
   "sdk": {
-    "version": "6.0.0",
+    "version": "8.0.0",
     "rollForward": "latestMajor",
-    "allowPrerelease": false
+    "allowPrerelease": true
```
Comment (lg2de) [inline global.json:5]: When using `rollForward:latestMajor` it doesn't really make sense to allow prereleases, isn't it?

Comment (Jan5366x) [reply global.json:5]: Correct, this is a left over from testing the .net 8.0 pre-release

Will be adjusted in:
PR: #8 
Commit: https://github.com/Jan5366x/MagicFileEncoding/pull/8/commits/30c173b39edbd24219932c7644f119d001253a55

**Pull request merged**